mobile/mobile-core/devices-service.ts

```diff
@@ -522,6 +523,11 @@ export class DevicesService implements Mobile.IDevicesService {
 	private clearCurrentDeviceDetectionIntervalFuture(): IFuture<void> {
 		return this.deviceDetectionIntervalFuture || Future.fromResult();
 	}
+
+	private getStartPackageActivity(framework?: string): string {
```

Comment:
This method appears to be Android-specific, you can consider moving it to some Android-specific class, like for example `AndroidApplicationManager`